docker-compose.yml

```
@@ -58,30 +68,26 @@ services:
     healthcheck:
       test: ["CMD", "wget", "-q", "--spider", "http://localhost:3223/health"]
       interval: 30s
-      timeout: 10s
+      timeout: 3s
       retries: 3
       start_period: 20m
-      start_interval: 30s
+      start_interval: 1s
 
   ensadmin:
     container_name: ensadmin
     image: ghcr.io/namehash/ensnode/ensadmin:latest
     ports:
       - "4173:4173"
     environment:
-      # Override ENSADMIN_PUBLIC_URL to point to docker compose ensnode-api
-      # Note: it must be a "public" URL accessible from a web browser (i.e. it cannot be a hostname exclusive to the internal docker network)
+      # Override environment variables to point to docker instances
       ENSADMIN_PUBLIC_URL: http://localhost:4173
-      # Override NEXT_PUBLIC_DEFAULT_ENSNODE_URLS to the docker compose ensnode-api
-      # Note: it must be a comma-separated list of URLs that are accessible from a web browser
-      # (i.e. it cannot be a hostname in the docker network)
-      NEXT_PUBLIC_DEFAULT_ENSNODE_URLS: http://localhost:42069
+      NEXT_PUBLIC_SERVER_CONNECTION_LIBRARY: http://localhost:4334
```

```suggestion
      NEXT_PUBLIC_SERVER_CONNECTION_LIBRARY: http://ensapi:4334
```

Is that ok? Please ignore if not relevant

---

unfortunately not possible here — because these urls must be accessible on the host (it's the browser that uses that `http://localhost:4334` url to talk to ensapi, not the docker container) it has to be host-accessible, and i don't think the docker hostname's work on the host by default